test/workspace.js

```diff
@@ -249,4 +249,61 @@ describe('Workspace', function() {
       });
     });
   });
+
+  describe('Creating 2 projects from template', function() {
```

Comment:
`describe('Workspace.createFromTemplate(...)', function(...)` like the other descriptions above.